console/console-init/ui/src/modules/iot-device/components/ReviewDevice/ReviewDevice.tsx

```diff
@@ -83,15 +105,65 @@ const ReviewDevice: React.FunctionComponent<IReviewDeviceProps> = ({
     return (
       <>
         <Text className={css(style.font_size_20)}>Gateways</Text>
-        {gateways && gateways?.length > 0
-          ? gateways.map(gateway => (
-              <span key="gateway">
-                <br />
-                {gateway}
-              </span>
-            ))
-          : "--"}
         <br />
+        <Split>
```

Comment:
I would like to suggest create generic function for render SplitItems and resuse that function for other plcaes as well i.e. gatewayGroups, MemberOf etc.

    const renderSplitItems=(items:string[])=>{
      return(
     <Split>
          {items && items?.length > 0
            ? items.map(item => (
                <SplitItem
                  key={item}
                  className={css(style.split_right_margin)}
                >
                  {item}
                </SplitItem>
              ))
            : "--"}
        </Split>
    )
    }